docs/serverless-v2/03-02-security.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Security
+type: Details
+---
+
+To eliminate potential security risks when using lambdas, bear in mind these few facts:
+
+- Kyma doesn't run any security scans against lambdas and their images. Before you store any sensitive data in lambdas, consider the potential risk of data leakage.
```

Comment:
```suggestion
- Kyma does not run any security scans against lambdas or their images. Before you store any sensitive data in lambdas, consider the potential risk of data leakage.
```
Just because you don't use contractions later on. I would also use `or` with negation 🤔

Reply:
I'll remove contraction here 👍 but I would leave `and` as I think that is crucial that both are not scanned.